papaparse.js

```diff
@@ -415,6 +415,50 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
+	{
+		this._columnConfig = {};
+		this._isAllDynamic = false;
+		this._isBlackListOnly = true;
+
+		if (config === true)
+		{
+			this._isAllDynamic = true;
+			return;
+		}
+		else if (!config)
+		{
+			return;
+		}
+
+		if (config.__only instanceof Array)
+		{
+			this._isBlackListOnly = false;
+			config.__only.forEach(function(field)
+			{
+				this._columnConfig[field] = true;
+			}.bind(this));
+		}
+		else
+		{
+			this._isBlackListOnly = !!config.__except;
+			this._columnConfig = config;
```

Comment:
Which is the objective of this line? For me the column config must be always a dict containing the expected only and the except attributes. Nothing else.